### PR TITLE
banner view implemented

### PR DESCRIPTION
**Pull request by iamavishkar**
## JIRA Ticket(s) if any

* [MOB-XXXX](https://iterable.atlassian.net/browse/MOB-XXXX)

## Test Steps

### CODE REVIEW

**Comment by Sagarpatel14** [comment]
image
image

File: src/components/banner/index.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+              order: 2;
+              max-width:75%;
+            }
+            .banner-container > div:last-child {
+              order: 1;
+              margin-bottom: 0;
+            }
+          }
+        `}
+      </style>
```
Comment (mprew97) [inline src/components/banner/index.tsx:106]: i'd really like to find a decent library to implement granular css styles like this. can this not be passed in directly to the `style` props same as the others? As much as possible, would prefer to pull this out of the actual component tree.

**Review by mprew97 — CHANGES_REQUESTED** [review]
We need to pull this into a new page in the react example app `/components` and render it there so it can be seen/used in real time.

**Comment by iamavishkar** [comment]
request you to please review the PR as it has been updated.

**Comment by mprew97** [comment]
> request you to please review the PR as it has been updated.

Ty sir!

We should be able to just import the `Banner` component in `react-example` as a user would, as opposed to duplicating it. Can we do that and also have an example of it being displayed on a page somewhere in the example? Like an actual live component example?

**Comment by devcsomnicg** [comment]
Hi @mprew97 here is the video for the same: https://drive.google.com/file/d/1ifip1OplImM9m2q-kOXGwxNwenqJXkCC/view?usp=sharing

**Review by mprew97 — APPROVED** [review]

**Comment by mprew97** [comment]
Once the build issues are resolved, we can merge this.

**Comment by mprew97** [comment]
Hi @devcsomnicg can we resolve the conflict here?

**Pull request merged**